salt/modules/pillar.py

```diff
@@ -62,6 +63,24 @@ def get(key,
             then merging will be skipped and the retrieved value will be
             returned. Earlier releases raised an error in these cases.
 
+    deep_merge_lists : ``None``
```

Comment:
This should just be called ``merge_lists``, and we don't typically put the default version in the docstring when it is ``None``. Also, we could be a lot more concise here and make the documentation much easier to understand. For example:

```
        If specified, this argument overrides the
        :conf_minion:`pillar_merge_lists` minion config option.

        .. note::
            This argument is ignored when ``merge`` is set to ``False``.
```

The ``:conf_minion:`` bit will link the user to the minion documentation which will explain how that option works, so we don't need to bother with re-explaining it here.